Registration-bootstrap.html

```diff
@@ -0,0 +1,81 @@
+        integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link rel="stylesheet" href="app.css">
+
+</head>
+<body>
+    <div class="container">
+        <h1 class="display-2">Register</h1>
+        <form action="nowhere">
+            <div class="form-row">
+            <div class="form-group col-md-6">
+                    <label for="first name">*First Name</label>
+                    <input type="first name" class="form-control" id="first name" placeholder="Enter your name">
+            </div><div class="form-group col-md-6">
+                    <label for="last name">*Last Name</label>
+                    <input type="last name" class="form-control" id="last name" placeholder="Enter your last name">
+            </div><div class="form-group col-md-7">
+                    <label for="last name">Nickname</label>
+                    <input type="nickname" class="form-control" id="nickname" placeholder="Enter your nickname (optional)">
+            </div>
+            
+            <div class="form-group col-md-6">
+                <label for="email">*Email Address</label>
+                <input type="email" class="form-control" id="email" placeholder="Enter your email adress">
+            </div>
+            <div class="form-group col-mid-6">
+                    <label for="password">*Password</label>
+                    <input type="password" class="form-control" id="password" placeholder="Enter your password">
+                </div>
+                <div class="form-group col-mid-6">
+                    <label for="repeat password">*Repeat Password</label>
+                    <input type="repeat password" class="form-control" id=" Repeat Password" placeholder="Repeat your password">
+                </div>
+            </div>
+        
+            <div class="form-group">
+                <label for="address">Street Address</label>
+                <input type="address" class="form-control" id="address" placeholder="Otto-Hahn-Street 110">
+            </div>
+            <div class="form-row">
+            <div class="form-group col-md-6">
+                <label for="city">City </label>
+                <input type="city" class="form-control" id="city">
+            </div>
+                
+                <div class="form-group col-md-3 col-6">
+                    <label for="zip">Zip Code </label>
+                    <input type="text" class="form-control" id="zip">
+                </div>
+            </div>
+            <div class="form-group"></div>
+                  <div class="custom-control custom-checkbox">
+                    <input type="checkbox" class="custom-control-input" id="tos">
+                    <label class="custom-control-label" for="tos">Sign Me Up</label>
+                </div>
+            </div>
+               
+            <button type="submit" class="btn btn-success">Register</button>
+          
+        </form>
+    </div>

```

Comment:
Between the code lines, 68 and 69 you should paste the  content of the entire footer  from the index.html file  so that we have the footer on the registration UI